FX/bryanchi_FX Devices.lua

```diff
@@ -0,0 +1,24 @@
+-- @description FX Devices
+-- @author Bryan Chi
+-- @version 1.0beta
+-- @changelog
+--   --------------------------==  declare Initial Variables & Functions  ------------------------
+--       VersionNumber = 'Dec 23 2022 - bug fix 3 '
+--       FX_Add_Del
+-- @provides
+--   . > BryanChi_FX_Devices.lua
+--   bryanchi_FX Devices/FXD MainScript.lua
+--   [data] bryanchi_FX Devices/IconFont1.ttf
```

Comment:
Data type means the file is installed in the subfolder of that name instead of under Scripts. Is this correct?

Reply:
Oh it's meant to be in the same folder as the Main Script, which type should I choose if it's a font file?

Reply:
Set it to "Script" to be installed in the same folder (`<resource path>/ReaTeam Scripts/FX/brianchi_FX Devices/IconFont1.ttf`).

The main script will be `<resource path>/ReaTeam Scripts/FX/BrianChi_FX Devices.lua`, so it will have to do `CreateFont(path .. '/BrianChi_FX Devices/IconFont1.ttf')`.

(Renaming the folder from brianchi to BrianChi is possible but cumbersome using the package editor, I can rename it for you more easily with git before releasing if you prefer the capitalized path.)